.test-infra/jenkins/job_LoadTests_GBK_Java_spark_structured_streaming.groovy

```diff
@@ -0,0 +1,208 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import CommonTestProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+
+def loadTestConfigurations = { mode, isStreaming, datasetName ->
```

Comment:
Not sure if this should be in separate file or together with other test cases in  `job__LoadTests_GBK_Java.grooy`

Reply:
@mwalenia could you confirm?

Reply:
as the java test file contained only dataflow tests, I separated spark but sure I can merge. Waiting for the answer

Reply:
I think we can split the job definitions runner-wise into files, that way the files won't get too bloated

Reply:
I'm fine with both options then.